collection-scripts/gather_ceph_resources

```diff
@@ -257,7 +257,6 @@ for ns in $namespaces; do
                     printf "Collecting image status for: %s/%s\n" "${bp}" "${image}"
                     timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd status $image --pool $bp" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-status-"${image}"-debug.log
                     printf "Collecting snap info for: %s/%s\n" "${bp}" "${image}"
-                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log
```

Comment:
why we have removed this one?

Reply:
in next line we are collecting same details with proper format, so it's just as duplication.

Reply:
But why was it there in the first place?

Reply:
not sure, what added it.

Reply:
It looks like there was actually a reason for adding the `--format=json` version: https://github.com/red-hat-storage/ocs-operator/commit/08d7f78e1db1c0414526a27288313be8ffb6be60 Please either return the line you removed or  confirm with Ilya that it's effectively redundant. I personally don't like to try and parse JSON if I don't need to.

Reply:
Similarly, if you add it back in be sure to add it for the rns block below as well.

Reply:
if we have the json formatted output, we don't need this. IMO, we can remove this.